packages/hardhat/contracts/strategy/CrabStrategyV2.sol

```diff
@@ -199,11 +199,14 @@ contract CrabStrategyV2 is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Own
         uint256 _wSqueethToMint,
         uint256 _crabSharesToMint,
         uint256 _timeAtLastHedge,
-        uint256 _priceAtLastHedge
+        uint256 _priceAtLastHedge,
+        uint256 _strategyCap
```

Comment:
this doesn't need to be passed in as a param right? Can just be the msg.value?

Reply:
comment above on this, lmk your thoughts